types/styled-components/test/index.tsx

```diff
@@ -336,6 +336,16 @@ const SomeButton: React.FC = () => <Button type="submit">I am a button</Button>;
  * withConfig
  */
 
+/**
+ * displayName
+ */
+
+styled('div').withConfig({
+    displayName: "TestDisplayName",
```

Comment:
Could you ask styled-components if that is a public option? It's not mentioned in their documentation so it may not be intended for public usage.

Reply:
I have [opened an issue](https://github.com/styled-components/styled-components-website/issues/782) to ask if this is intended for public usage, and if so if the documentation should be updated to show this. 

I did also find [a reference to this argument in a documentation example](https://styled-components.com/docs/tooling#namespace) of what the transpiled code looks like, though it is still unclear from that example on if this is intended to be configured by developers or if it is only intended for internal use by transpiled code.

Reply:
It might be possible that this is not intended for public usage considering the property is used by babel plugin they control.

Reply:
Yeah, agreed. Let's wait and see if there is a response to the documentation question.